scanomatic/models/factories/settings_factories.py

```diff
@@ -176,7 +176,8 @@ class ApplicationSettingsFactory(AbstractModelFactory):
     @classmethod
     def create(cls, **settings) -> settings_models.ApplicationSettingsModel:
         cls.populate_with_default_submodels(settings)
-
+        if 'versions' in settings:
+            del settings['versions']
```

Comment:
hmm?

Reply:
The model has a field version but it may not be supplied upon creation and should instead always be constructed from the current default creation inside the model's init function. We might be able to later do away completely with this versions thing that is responsible for some backwards compatibility, but I dare not now.

Reply:
I see. And it is `versions` not `version`? (In your explanation you seem to imply it is a single version per model.)

Reply:
yes well it could become whatever, but I'd rather keep it and make an issue for investigating removing it.

Reply:
https://github.com/Scan-o-Matic/scanomatic-standalone/issues/92